internal/experiment/generation/stormforger.go

```diff
@@ -97,12 +114,22 @@ func (s *StormForgerSource) Update(exp *redskyv1beta1.Experiment) error {
 		pod.Containers[0].Env = append(pod.Containers[0].Env, corev1.EnvVar{Name: "TARGET", Value: ingressURL})
 	}
 
+	// Add a reference to the access token
+	pod.Containers[0].Env = append(pod.Containers[0].Env, corev1.EnvVar{
```

Comment:
Is there a reason to handle this outside of the env setup earlier in this function ( where we set TITLE and TEST_CASE )

Reply:
Nope, just putting the credential var at the bottom of the list so it looks nice.